### ISSUE TITLE
Production paths are relative to docroot, not current folder

### ISSUE THREAD

**Issue by bobpaul** [Awesome-Technologies/synapse-admin#11]
I did a `yarn build` and then tried to serve the contents of the build folder with Apache webserver. I wanted to serve it from `chatserver.example.com/admin` but that results in 404s when trying to load `chatserver.example.com/static/js/main.78b12af2.chunk.js` which is no surprise... the files are in `chatserver.example.com/admin/static/js/main.78b12af2.chunk.js`

This would be solved if the path in the `index.html` lacked the initial slash. Eg:

```
<script src="static/js/2.2e1c8a1a.chunk.js">
```

**Comment by awesome-manuel** [comment]
You can rewrite the request using Apache. We typically do that using Traefik.
But you're right, the change will most probably not break anything.

**Comment by thomasdondorf** [comment]
I would also be interested in this. Seems like it's currently impossible to run the application in a directory.

Our scenario is:
- Synapse is running on `example.com`
- Synapse Admin is running on `example.com/admin`

Rewriting only works if Synapse Admin is the only application running on the domain.

**Comment by dklimpel** [comment]
I had the same problem.
I have found a solution for me: https://stackoverflow.com/questions/42686149/cant-build-create-react-app-project-with-custom-public-url#54092796

```PUBLIC_URL="/admin" npm start```

**Comment by dklimpel** [comment]
You can also make a build and use your own web server like Apache or Nginx.
```
npm install
PUBLIC_URL="/admin" npm run build
```
Now you have to link your server to `build` folder.

**Comment by awesome-manuel** [comment]
Should be fixed with 1002b6464a1093b000ebf1d3e9b26cb04526ff11

See also https://github.com/facebook/create-react-app/issues/527
